### DISCUSSION TITLE
Bug in Dit that could affect Pixart

### DISCUSSION THREAD

**Discussion by Bigfield77** [PixArt-alpha/PixArt-alpha#96] [General]
Hello,

I am not sure if this is relevant: https://github.com/facebookresearch/DiT/issues/41#issuecomment-1974823490

Basically a user asked a question about why classifier-free guidance was only applied to 3 dimension:
![image](https://github.com/PixArt-alpha/PixArt-alpha/assets/108305548/759c5d82-8e7d-4bac-95dd-c97c81b85d63)

According to the reply the issue would also be present in Pixart.
![image](https://github.com/PixArt-alpha/PixArt-alpha/assets/108305548/1e8f88a5-bbfd-4476-a440-5667a8d08ede)

I haven't checked due to my limited knowledge:)

cheers

**Comment by lawrence-cj** [comment]
Yah, we tried to change it from 3 to 4, which not influence much. And our default sampler is DPM-Solver, which don't have this bug issue.
